Tools/WebKitTestRunner/Configurations/WebKitTestRunner.xcconfig

```diff
@@ -31,6 +31,8 @@ PRODUCT_NAME = WebKitTestRunner;
 PRODUCT_BUNDLE_IDENTIFIER = com.apple.WebKit.WebKitTestRunner;
 GCC_ENABLE_OBJC_EXCEPTIONS = YES;
 OTHER_LDFLAGS[sdk=macosx*] = $(inherited) -lWebKitTestRunner -framework Carbon -framework Cocoa -framework JavaScriptCore -framework UniformTypeIdentifiers -framework WebKit;
+OTHER_LDFLAGS[sdk=iphone*] = $(inherited) -framework JavaScriptCore;
+OTHER_LDFLAGS[sdk=xr*] = $(inherited) -framework JavaScriptCore;
```

Comment:
Shouldn't we also do this for watchOS and tvOS to avoid breaking local builds?

Reply:
Maybe we should just link it unconditionally?

Reply:
Even if it's truly not necessary, I think unconditional linkage would be okay.